### PR TITLE
api/server: add project id/name in ssh keypair response

### LINKED ISSUE #7096: Unable to delete SSH Key as User within a project
##### ISSUE TYPE

 * Bug Report

##### CLOUDSTACK VERSION
4.17+ (maybe previous as well)

##### SUMMARY
Unable to delete SSH Key as User within a project

##### STEPS TO REPRODUCE
- Create a project as admin
- Add a user to the project
- Log in as the user and enter the project
- Create an ssh key as user within the project
- Verify the key is created successfully
- Try to delete the key
- `Request failed. (431) : ProjectId and account/domainId can't be specified together`

http://whatever:8080/client/api/?name=key&account=PrjAcct-proj-1&domainid=cfa951cf-909c-11ed-a946-1e008400027e&command=deleteSSHKeyPair&response=json&projectid=4e96a78d-2581-4d94-b1cd-6e7b392a1a29

##### EXPECTED RESULTS
The SSH Key is deleted

##### ACTUAL RESULTS
The account and domain id of the key is passed along with the project ID
Only the projectid should be specified in the API call

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This fixes #7096

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]

File: api/src/main/java/org/apache/cloudstack/api/response/SSHKeyPairResponse.java [hunk 1]
```diff
@@ -46,6 +46,14 @@ public class SSHKeyPairResponse extends BaseResponseWithAnnotations {
     @SerializedName(ApiConstants.DOMAIN) @Param(description="the domain name of the keypair owner")
     private String domain;
 
+    @SerializedName(ApiConstants.PROJECT_ID)
+    @Param(description = "the project id of the keypair owner")
+    private String projectId;
+
+    @SerializedName(ApiConstants.PROJECT)
+    @Param(description = "the project name of the keypair owner")
+    private String projectName;
```
Comment (DaanHoogland) [inline api/src/main/java/org/apache/cloudstack/api/response/SSHKeyPairResponse.java:55]: why do we need both in the response?

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by davidjumani — APPROVED** [review]

**Review by davidjumani — APPROVED** [review]
Tested, fixes the issue

**Comment by weizhouapache** [comment]
> Tested, fixes the issue

good thanks @davidjumani for testing

**Pull request merged**